tools/webpack.config.prod.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
```

Comment:
It would make sense to move similar parts of "dev" and "prod" to the webpack.config.base.js